server/pkg/thread/comment_test.go

```diff
@@ -10,14 +11,30 @@ func TestComment_CommentType(t *testing.T) {
 	cid := NewCommentID()
 	uid := NewUserID()
 	c := "xxx"
+	tim, _ := time.Parse(time.RFC3339, "2021-03-16T04:19:57.592Z")
 
 	got := Comment{
-		id:      cid,
-		author:  uid,
-		content: c,
+		id:        cid,
+		author:    uid,
+		content:   c,
+		createdAt: tim,
 	}
 
 	assert.Equal(t, cid, got.ID())
 	assert.Equal(t, uid, got.Author())
 	assert.Equal(t, c, got.Content())
+	assert.Equal(t, tim, got.CreatedAt())
+}
+
+func TestComment_SetContent(t *testing.T) {
+	c := "xxx"
+	got := Comment{}
+
+	got.SetContent(c)
+	assert.Equal(t, got.Content(), c)
```

Comment:
```suggestion
	comment := Comment{}
	comment.SetContent(c)
	assert.Equal(t, "xxx", comment.content)
```

- "got" = "result". In this case it is not proper name.
- Note the order: `assert.Equal(t, EXPECTED, ACTUAL)`
- When testing only SetContent, it is best to use as few other methods as possible. When the implementation of other methods is changed, this test case will also be affected.

Reply:
Done!